packages/ui/playground/vscode-theme.css

```
@@ -0,0 +1,159 @@
+    --vscode-descriptionForeground: #9d9d9d;
+    --vscode-disabledForeground: #6e6e6e;
+    --vscode-textLink-foreground: #4daafc;
+    --vscode-panel-border: #3c3c3c;
+    --vscode-widget-border: #3c3c3c;
+    --vscode-focusBorder: #007fd4;
+    --vscode-button-background: #0e639c;
+    --vscode-button-hoverBackground: #1177bb;
+    --vscode-testing-iconPassed: #4ec9b0;
+    --vscode-editorWarning-foreground: #cca700;
+    --vscode-editorError-foreground: #f44747;
+    --vscode-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)